release/util.bzl

```diff
@@ -8,7 +8,7 @@ inputs = {
     "sdk_config": ":sdk-config.yaml.tmpl",
     "install_sh": ":install.sh",
     "install_bat": ":install.bat",
-    "sandbox_logback": "//ledger/sandbox-common:src/main/resources/logback.xml",
+    "sandbox_logback": "//ledger/sandbox-on-x:src/app/resources/logback.xml",
```

Comment:
I am not sure I am doing the right thing here. Where exactly this file is used? I have deleted an original one from `sandbox-common`

Reply:
I think it's fine to delete. All of the other `*-logback.xml` files under `$DAML_HOME/sdk/$VERSION/daml-sdk` are used either directly in the [Daml Assistant code](https://github.com/digital-asset/daml/tree/main/daml-assistant/daml-helper/src/DA/Daml/Helper) or the [sdk config](https://github.com/digital-asset/daml/blob/main/release/sdk-config.yaml.tmpl).

Reply:
done